common/src/main/java/net/xolt/freecam/config/CollisionWhitelist.java

```diff
@@ -8,16 +8,14 @@
 public class CollisionWhitelist {
 
     private static final Collection<Class<? extends Block>> transparentWhitelist = List.of(
-            AbstractGlassBlock.class,
+            TransparentBlock.class,
```

Comment:
`TransparentBlock` is extended by `GrateBlock`, `StainedGlassBlock`, & `TintedGlassBlock`.

Don't think we want `GrateBlock` in the whitelist (it's not fully transparent _enough_ that you should be able to no-clip through it), so should probably be:
```suggestion
            StainedGlassBlock.class,
            TintedGlassBlock.class,
```

This was probably already an issue, but the mapping change highlighted it.

Reply:
@MattSturgeon This actually broke Ignore Transparent for plain glass blocks, it looks like TintedGlass is something else. 

> it's not fully transparent enough that you should be able to no-clip through it

We do include iron bars in the same category, which have a similar transparency level to grates. I think they could be included

Reply:
> This actually broke Ignore Transparent for plain glass blocks, it looks like TintedGlass is something else.

Interesting. Good catch. Perhaps yarn got the name wrong, wouldn't be the first time. Guess I should've tested.

> We do include iron bars in the same category

I'd forgotten about that.  
That's one of the limitations of `CollisionWhitelist` (currently) only using the class. IIRC iron bars and glass panes share the same class, the only way to tell them apart is block data.

Might look into using block id instead, maybe allowing the user to create their own whitelist/blacklist...